individual/services.py

```diff
@@ -82,3 +86,34 @@ def update(self, obj_data):
     @register_service_signal('group_individual.delete')
     def delete(self, obj_data):
         return super().delete(obj_data)
+
+
+class GroupFromMultipleIndividualsService(BaseService):
+    OBJECT_TYPE = Group
+
+    def __init__(self, user, validation_class=GroupValidation):
+        super().__init__(user, validation_class)
+
+    @check_authentication
+    @register_service_signal('group_from_multiple_individuals_service.create')
+    def create(self, obj_data):
+        try:
+            with transaction.atomic():
+                individual_ids = obj_data.pop('individual_ids')
+                group = GroupService(self.user).create(obj_data)
+                group_id = group['data']['id']
+                individual_ids_list = [GroupIndividualService(self.user).create({'group_id': group_id,
+                                                                                 'individual_id': individual_id})
+                                       for individual_id in individual_ids]
+                group_and_individuals_dict = {**group, 'individuals': individual_ids_list}
```

Comment:
Do we need this kind of output? Mixing group output with id's seems excessive. You can still access individuals through Group object. If you're worried about efficiency (like returning group here and fetching once again individuals later on), you can use select_related().

Also output should have format of `output_result_success`.

Reply:
now it is in detail of **output_result_success**